tikv/tikv
# importer: auto split when import end

Signed-off-by: Little-Wallace <bupt2013211450@gmail.com>



### What problem does this PR solve?


Problem Summary:
If region split during import, the sst_importer::write rpc has to retry and it may hurt performance of import. And lightning may import a lot of data to one region, this may cause a large region. So I hope TiKV can split the region when import end.


### What is changed and how it works?

TiKV will check the import mode of importer before scheduling a split-check task.  If the approximate size is too large, TiKV will schedule a split-check task.

### Related changes

- PR to update `pingcap/docs`/`pingcap/docs-cn`:
- PR to update `pingcap/tidb-ansible`:
- Need to cherry-pick to the release branch

### Check List 

Tests 

- Unit test
- Manual test (add detailed scripts or steps below)

Side effects

- Performance regression
    - Consumes more CPU.  There may be a lot of split-check task after import task ending, and these tasks will cost more CPU.

### Release note 
- No release note.
